bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/ComponentSensor.java

```diff
@@ -27,6 +30,7 @@
 @NonNullByDefault
 public class ComponentSensor extends AbstractComponent<ComponentSensor.ChannelConfiguration> {
     public static final String sensorChannelID = "sensor"; // Randomly chosen channel "ID"
+    private static final List<String> triggerObjectIDs = Arrays.asList(new String[] { "click", "action" });
```

Comment:
Why do you explicitly create the `String[]`? The `asList` method has varags paramters

Reply:
Sorry, I wasn't sure about what type of arguments the ```asList``` function would accept, so I looked up an example on the Internet. I will change it, thanks :+1: